doc/developer/building-frr-for-centos6.rst

```diff
@@ -26,7 +26,7 @@ CentOS 6 restrictions:
 -  MPLS is not supported on ``CentOS 6``. MPLS requires Linux Kernel 4.5
    or higher (LDP can be built, but may have limited use without MPLS)
 -  Zebra is unable to detect what bridge/vrf an interface is associated
-   with (IFLA\_INFO\_SLAVE\_KIND does not exist in the kernel headers,
+   with (IFLA\_INFO\_SECONDARY\_KIND does not exist in the kernel headers,
```

Comment:
this is changing what the kernel header reports but is now factually incorrect :(.

Reply:
Missed that one. Will fix.